scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -626,20 +626,22 @@ trait CliTestBehavior { this: AbstractCliTest =>
           |/foo.scala
           |object    A { }
           |""".stripMargin
+      val dir = string2dir(input)
       noArgTest(
-        string2dir(input),
+        dir,
         input,
         Seq(Array("--test")),
         assertExit = { exit => assert(exit.is(ExitCode.TestError)) },
         assertOut = out => {
+          val filename = "b" + dir + "/foo.scala\n"
+          val desiredOutput = filename + """|@@ -1,1 +1,1 @@
+            |-object    A { }
+            |+object A {}
+            |error: --test failed
+            |To fix this ...""".stripMargin
           assertContains(
             out,
-            """|foo.scala-formatted
```

Comment:
why not just replace this single line with:
```
  s"""b$dir/foo.scala
```
and remove the `desiredOutput` variable?